src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.ClientCertProvider.cs

```diff
@@ -0,0 +1,209 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography.X509Certificates;
+using System.Security.Cryptography;
+using Microsoft.Win32.SafeHandles;
+
+using CURLoption = Interop.libcurl.CURLoption;
+using CURLcode  = Interop.libcurl.CURLcode;
+using CURLINFO = Interop.libcurl.CURLINFO;
+using SslClientCertCb = Interop.libcurl.client_cert_cb;
+using SafeSslContextHandle = Interop.libssl.SafeSslContextHandle;
+
+namespace System.Net.Http
+{
+    internal partial class CurlHandler : HttpMessageHandler
+    {
+        [Conditional(VerboseDebuggingConditional)]
+        private static void IgnorePeerCert(EasyRequest easy)
+        {
+            easy.SetCurlOption(CURLoption.CURLOPT_VERIFYPEER, 0L);
```

Comment:
Doesn't this change behaviors?  Seems like something we shouldn't be doing in conditionally-compiled code